internal/testhelpers/session.go

```diff
@@ -33,6 +33,48 @@ func NewSessionLifespanProvider(expiresIn time.Duration) *SessionLifespanProvide
 	return &SessionLifespanProvider{e: expiresIn}
 }
 
+type SessionPrivilegedMaxAgeProvider struct {
+	e time.Duration
+}
+
+func (p *SessionPrivilegedMaxAgeProvider) SelfServiceFlowSettingsPrivilegedSessionMaxAge(ctx context.Context) time.Duration {
+	return p.e
+}
+
+func NewSessionPrivilegedMaxAgeProvider(privilegedMaxAge time.Duration) *SessionPrivilegedMaxAgeProvider {
+	return &SessionPrivilegedMaxAgeProvider{e: privilegedMaxAge}
+}
+
+type SessionLifespanAndPrivilegedMaxAgeProvider struct {
+	l *SessionLifespanProvider
+	p *SessionPrivilegedMaxAgeProvider
+}
```

Comment:
I joined the lifespan and privileged max age providers to avoid having to essentially pass the same config in twice. But the name of this provider is large and unwieldy 🤔